src/graphql/mstable/query.graphql

```diff
@@ -65,3 +65,19 @@ query UserLockups($account: Bytes, $hasAccount: Boolean!) @api(name: mstable) {
     }
   }
 }
+
+query AllUserLockups($minLockTime: BigInt!) @api(name: mstable) {
+  userLockups(
+    where: { lockTime_gte: $minLockTime }
+    first: 1000
```

Comment:
PS we know this won't do in the long term, but for now there are only 385 stakers